src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html

```diff
@@ -239,12 +243,58 @@ <h2>Request for judgment by default</h2>
           {{ ccj.amount.remainingAmount }}
         </td>
       </tr>
+
+      {% if ccj.repaymentPlanContent.repaymentOption == "Immediately" %}
       <tr>
         <td>When you want the defendant to pay:</td>
         <td>
-          {{ ccj.repaymentOption }}
+          {{ ccj.repaymentPlanContent.repaymentOption }}
```

Comment:
This is duplicated 3 times. Please print `When you want the defendant to pay:` row just once as it comes first. After that print remaining rows depending on selected repayment option.

REQ

Reply:
Its only used 2 times, Instalments doesn't have "When you want the defendant to pay" (if you are checking prototype its wrong, it should be "How do you want the defendant to pay" not "when").

Also, "When you want the defendant to pay" comes second for "By a set date".

But because its same text 2 times, i will move that text to macro. I think that's probably better...you agree?